### PR TITLE
Runs linter to fix lint errors

### PR DESCRIPTION
**Pull request by seungpark**
Updated the `npm format` script to `biome lint --write` - this will only write the safe fixes, and return the unsafe fixes to be manually applied.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "build:util": "pnpm --filter util build",
     "build:git-changed-files": "pnpm build:util && pnpm --filter git-changed-files build",
-    "lint": "biome ci",
-    "format": "biome format --write"
+    "lint": "biome lint",
```
Comment (seungpark) [inline package.json:9]: is there a preference between `ci` and `lint`?

File: extensions/offline-snooty/src/convertGatsbyToHtml/fileHandler.ts [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
         '/',
       );
       node.setAttribute('href', targetHref);
     } else if (
       node instanceof HTMLImageElement &&
-      node['src'].startsWith('/')
+      node.src.startsWith('/')
     ) {
-      node['src'] = (prefix + node['src']).replace(/\/+/, '/');
+      node.src = (prefix + node.src).replace(/\/+/, '/');
```
Comment (biancalaube) [inline extensions/offline-snooty/src/convertGatsbyToHtml/fileHandler.ts:19]: is this only going to fix the linting errors in this extension or all extensions? If its only for this extension would there be a way to fix it for all current/future extensions?

Comment (seungpark) [reply extensions/offline-snooty/src/convertGatsbyToHtml/fileHandler.ts:19]: I ran the `npm run format` in the root folder! it should update all extensions, see example [here](https://github.com/mongodb/netlify-extensions/pull/40/files#diff-e99d1403bc297b4375153b18caf69e809a51369eaf0320ceb50a1a6c01f44637)

File: package.json (hunk 1 above)
Comment (anabellabuckvar) [reply package.json:9]: IMO, `lint` feels more intuitive here since we're calling the script `lint`

File: package.json [hunk 3]
```diff
@@ -6,8 +6,8 @@
...
+    "format": "biome lint --write"
```
Comment (anabellabuckvar) [inline package.json:10]: Why change this from `format` to `lint`?

Comment (seungpark) [reply package.json:10]: `format` and `lint` are returning different results. perhaps due to the biome.json config? cc @branberry . i would like to keep the npm commands to keep the same check with `format` overwriting and `lint` just checking

File: package.json (hunk 1 above)
Comment (seungpark) [reply package.json:9]: we can always pass in more options with `npm run lint -- --write` but that would be the same as `npm run format` in this case

**Review by branberry — APPROVED** [review]

**Comment by branberry** [comment]
Thank you for doing this!

File: package.json (hunk 3 above)
Comment (seungpark) [reply package.json]: discussed offline with @branberry ! we included both! lint and formatting does two different things, and CI encapsulates both. so now, `npm run format` will fix both linting and formatting, and `npm run lint` will check for both

**Pull request merged**